operator/src/main/java/oracle/kubernetes/operator/helpers/JobStepContext.java

```diff
@@ -558,9 +558,11 @@ protected List<V1Container> getContainers() {
   protected List<V1Volume> getFluentdVolumes() {
     List<V1Volume> volumes = new ArrayList<>();
     Optional.ofNullable(getDomain())
-            .map(DomainResource::getFluentdSpecification)
-            .ifPresent(c -> volumes.add(new V1Volume().name(FLUENTD_CONFIGMAP_VOLUME)
-                    .configMap(new V1ConfigMapVolumeSource().name(FLUENTD_CONFIGMAP_NAME).defaultMode(420))));
+        .map(DomainResource::getFluentdSpecification)
+        .ifPresent(c -> volumes.add(new V1Volume().name(FLUENTD_CONFIGMAP_VOLUME)
```

Comment:
Do we have the same problem with fluentd configmap volume names?

Reply:
No, because the volume name only exists inside the container -- the volume is not an independent resource stored on the API server like a ConfigMap is.

Reply:
ok. thanks